docs/operator-manual/config-management-plugins.md

```diff
@@ -178,7 +178,7 @@ entrypoint. You can use either off-the-shelf or custom-built plugin image as sid
 containers:
 - name: my-plugin
   command: [/var/run/argocd/argocd-cmp-server] # Entrypoint should be Argo CD lightweight CMP server i.e. argocd-cmp-server
-  image: busybox # This can be off-the-shelf or custom-built image
+  image: registry.access.redhat.com/ubi8 # This can be off-the-shelf or custom-built image
```

Comment:
I don't see a reason we need to switch the example to use ubi8 also stick to a registry from RedHat. So I will decline this PR.

Reply:
As I said, I am open to switch this for another image.

But we should somehow make sure, that this missing `ca-certificates`  issue get's resolved. I literally spend a day trying to find out what was wrong, as I did everything as per documentation.